test/test_common/network_utility.cc

```diff
@@ -88,6 +88,13 @@ const std::string getLoopbackAddressUrlString(const Address::IpVersion version)
   return std::string("127.0.0.1");
 }
 
+const std::string getLoopbackAddressString(const Address::IpVersion version) {
+  if (version == Address::IpVersion::v6) {
+    return std::string("::1");
+  }
+  return std::string("127.0.0.1");
+}
+
 const std::string getAnyAddressUrlString(const Address::IpVersion version) {
   if (version == Address::IpVersion::v6) {
     return std::string("[::]");
```

Comment:
Various integration tests use brackets to form URLs from substitutions, e.g. `tcp://{{ ip_loopback_address }}::{{ some_port }}`. We need brackets in this case, as this is the URI format.